src/App.jsx

```diff
@@ -14,33 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABETICALLY = 'alphabetically';
+const SORT_BY_LENGTH = 'by length';
```

Comment:
this approach is ok in this case, but I believe it would be better to have it as an object, so these are stored together. We will have it a similar way with `enum` later on when we'll start using Typescript, so here's a good chance to practice it a bit. 

```suggestion
const SORT_BY =  {
  alphabet: 'alphabet',
  length: 'length',
};
```

And then you can reference it like `SORT_BY.alphabet`;